packages/web/src/components/remix-settings-modal/RemixSettingsModal.tsx

```diff
@@ -114,6 +165,123 @@ const RemixSettingsModal = ({
     onClose(trackId)
   }, [onClose, track, isInvalidTrack, url])
 
+  if (isPremiumContentEnabled) {
+    return (
+      <Modal
+        isOpen={isOpen}
+        onClose={onCloseModal}
+        wrapperClassName={styles.remixSettingsModalContainer}
+        dismissOnClickOutside
+      >
+        <ModalHeader
+          className={styles.remixSettingsModalHeader}
+          onClose={onCloseModal}
+          showDismissButton
+          dismissButtonClassName={styles.remixSettingsModalHeaderDismissButton}
+        >
+          <ModalTitle
+            title={messages.title}
+            icon={<IconRemix className={styles.remixSettingsModalTitleIcon} />}
+          />
+        </ModalHeader>
+        <ModalContent>
+          {isPremium && (
+            <div className={styles.disableInfo}>
+              <IconQuestionCircle className={styles.disableInfoIcon} />
+              <div>
+                <span>{messages.changeAvailabilityPrefix}</span>
+                <span>
+                  {'nft_collection' in (premiumConditions ?? {})
+                    ? messages.collectibleGated
+                    : messages.specialAccess}
+                </span>
+                <span>{messages.changeAvailabilitySuffix}</span>
+              </div>
+            </div>
+          )}
+          <div className={styles.toggleRow}>
+            <span className={cn({ [styles.remixDisabled]: isPremium })}>
+              {messages.markAsRemix}
+            </span>
+            <Switch
+              isOn={isRemix}
+              handleToggle={(e) => {
+                e.stopPropagation()
```

Comment:
Ideally we shouldn't do this ever - why is it necessary?

Reply:
yup dont need em. mustve been copy-paste-forget. good catch